deploy/sre-prometheus/100-machine-api.PrometheusRule.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-machine-api-node-mismatch
+    role: alert-rules
+  name: sre-machine-api-node-mismatch
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-machine-api-node-mismatch
+    rules:
+    - alert: NoNodeObjectForMachineWarning
+      expr: &promql machine_api_status == 0
+      for: 30m
```

```suggestion
      for: 15m
```
I assume provisioning a node might take 10 minutes based on what I've observed.  15 min seems a good time to warn.